src/forms/form-fields/multi-select.js

```diff
@@ -209,7 +213,9 @@ export default React.createClass({
     },
 
     _assignItems(items) {
-        if (this.props.referenceProperty === 'aggregationLevels') {
+        debugger;
```

Comment:
?

Reply:
@Birkbjo I knew we had a `debugger` statement... 

Also not sure why our eslint config lets that pass, but won't accept console.log? Thats probably something to look into. @jenniferarnesen